java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -225,4 +225,52 @@ public interface ListBaseCommands {
      * }</pre>
      */
     CompletableFuture<String[]> rpopCount(String key, long count);
+
+    /**
+     * Pops an element from the head of the first list that is non-empty, with the given keys being
+     * checked in the order that they are given.<br>
+     * Blocks the connection when there are no elements to pop from any of the given lists.
+     *
+     * @see <a href="https://redis.io/commands/blpop/">redis.io</a> for details.
+     * @apiNote <code>BLPOP</code> is a client blocking command, see <a
+     *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
+     *     Commands</a> for more details and best practices.
+     * @param keys The <code>keys</code> of the lists to pop from.
+     * @param timeout The <code>timeout</code> of blocking in seconds; <code>0</code> causes to block
```

Comment:
nit: same suggestions for the other blpop and brpop docs
```suggestion
     * @param timeout The number of seconds to wait for a blocking <code>BLPOP</code> operation to complete. A value of <code>0</code> will block indefinitely.
```